tock/hours/models.py

```diff
@@ -218,14 +218,21 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
     @classmethod
-    def get_most_recent_periods(cls, number_of_periods=1):
+    def get_most_recent_periods(cls, number_of_periods=1, number_of_results=None):
         """
         Return the most recent N completed reporting periods
         A reporting period is complete if it's end date
         has is today or in the past.
         """
         today = datetime.date.today()
-        return ReportingPeriod.objects.filter(end_date__lte=today).order_by('-start_date')[:number_of_periods]
+        periods = ReportingPeriod.objects.filter(end_date__lte=today)
+
+        if number_of_results:
+            range_end = number_of_periods + number_of_results
+            return periods.order_by('-start_date')[number_of_periods:range_end]
```

Comment:
I just realized that I think I need to subtract from the `number_of_periods` because of how the Django limit / offset logic works.

`[:5]` => 5th result
`[5:10]` => *6th* through 10th result

So if the user puts in "5 weeks" I think this will need to be `[4:5]` to catch the 5th result.